script.recap/addon.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.recap" name="Recap" version="1.0.0" provider-name="robwebset">
+	<requires>
+		<import addon="xbmc.python" version="2.20.0"/>
+		<import addon="script.module.beautifulsoup" version="3.2.1"/>
+		<import addon="script.module.requests" version="2.3.0"/>
+	</requires>
+  	<extension point="xbmc.python.script" library="default.py"/>
+	<extension point="xbmc.service" library="service.py" start="login"/>
+	<extension point="kodi.context.item" library="default.py">
+		<item>
+			<label>32001</label>
+			<visible>IsEmpty(Window(videos).Property("RecapHideContextMenu")) + [Container.Content(tvshows) | Container.Content(seasons) | Container.Content(episodes)]</visible>
```

Comment:
Might be clearer when reading, but it´s wrong semantically and doesnt work for edge cases. (example: movies can get listed on season level with "link-to-tvshow" feature)